src/ensembl/src/services/api-service.ts

```diff
@@ -23,6 +23,9 @@ type FetchOptions = {
   preserveEndpoint?: boolean;
```

Comment:
Generally, regarding the api-service, I think it will have to be updated in a way that it doesn't have a default `host` (and, by extension, does not have the `preserveEndpoint` option), because it looks likely that most requests will be sent to the current domain anyway; but this should probably be done in a separate PR.

Reply:
To have a rough idea, see the changes in the cicd-test draft PR: https://github.com/Ensembl/ensembl-client/pull/70/files#diff-ddff3a47f9c4b497baea0d3bb6d1a2bf

We will also leverage webpack dev server's proxy option for proxying requests to appropriate backend hosts in development, instead of encoding that into the api-service.